processor/src/test/java/org/mapstruct/ap/test/bugs/_2170/mapper/AddressMapper.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright MapStruct Authors.
+ *
+ * Licensed under the Apache License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package org.mapstruct.ap.test.bugs._2170.mapper;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.ap.test.bugs._2170.dto.AddressDto;
+import org.mapstruct.ap.test.bugs._2170.entity.Address;
+import org.mapstruct.factory.Mappers;
+
+/**
+ * @author Filip Hrisafov
+ */
+//CHECKSTYLE:OFF
```

Comment:
Why is this needed?

Reply:
Checkstyle was complaining about an interface with no methods